### DISCUSSION TITLE
Did anyone tried the swift-srgan model?

### DISCUSSION THREAD

**Discussion by FNsi** [chaiNNer-org/chaiNNer#2251] [General]
Seems nobody mention it?

The origin model only training with 

_Number of High-Resolution images 800 (DIV2K) + 2650 (Flickr2K)_

And gain acceptable PSNR & SSIM score within 1 mb size.
......

**Comment by RunDevelopment** [comment]
I don't think so, at least [none have been published](https://openmodeldb.info/?t=arch%3Aswift-srgan). You might want to ask that question on [the EnhanceEverything discord server](https://discord.gg/GXqcVbQG). The people in there will know more.

**Reply by FNsi** [reply]
I think the biggest problem is there's even no official inference release...

Only train.py, but his code offer enough detail to running it.

I might be wrong anyway😂

**Reply by FNsi** [reply]
Ah, I have to confirm that swiftsrgan is not good enough for upscale, it always try to add much detail but make the pic looks dirty...

Btw I do test more esr model to compare, 

None of them could actually good handle this [Capo Caccia Coastline](https://10wallpaper.com/wallpaper/3840x2400/2306/Cave_Mountains_Coastline_Capo_Caccia_Alghero_Bing_4K_3840x2400.jpg)

Without enough training for depth.
